src/index.html

```diff
@@ -7,6 +7,289 @@
+        <div class="consultation__info">
+          <span class="consultation__info-descr">Free consultation with exceptional quality</span>
+          <span class="consultation__info-tel">Just one call away: <a href="tel:+84 1102 2703">+84 1102 2703</a></span>
+        </div>
+        <button href="#" class="consultation__btn">Get your consultation</button>
+      </section>
+<!-- ------------------ PROJECTS --------------------- -->
+      <section class="projects">
+        <h2 class="projects__title">Projects</h2>
+        <div class="projects__tabs-body">
+          <div class="projects__tabs">
+            <a href="#" class="projects-tab">All</a>
+            <a href="#" class="projects-tab">Commercial</a>
+            <a href="#" class="projects-tab">Residencial</a>
+            <a href="#" class="projects-tab">Other</a>
+          </div>
+          <div class="projects__slider">
+            <div class="projects__slider__cards">
+
+              <div class="projects-card">
+                <img src="./assets/images/wildstone-infra.png" alt="Wildstone Infra Hotel picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Wildstone Infra Hotel</span>
+                  <span class="projects-card__descr-address">2715 Ash Dr. San Jose, South Dakota</span>
+                </div>
+              </div>
+
+              <div class="projects-card">
+                <img src="./assets/images/wish-stone.png" alt="Wish Stone Building picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Wish Stone Building</span>
+                  <span class="projects-card__descr-address">2972 Westheimer Rd. Santa Ana, Illinois</span>
+                </div>
+              </div>
+
+              <div class="projects-card">
+                <img src="./assets/images/parkinston.png" alt="Mr. Parkinston's House picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Mr. Parkinston&#39;s House</span>
+                  <span class="projects-card__descr-address">3517 W. Gray St. Utica, Pennsylvania</span>
+                </div>
+              </div>
+
+              <div class="projects-card">
+                <img src="./assets/images/oregano-height.png" alt="Oregano Height picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Oregano Height</span>
+                  <span class="projects-card__descr-address">2464 Royal Ln. Mesa, New Jersey</span>
+                </div>
+              </div>
+
+            </div>
+            <div class="projects__slider__navigation">
+              <button class="description-slider__button btn-next projects__slider-btn">Next
```

Comment:
`btn`, and probably `btn--next` and `btn--prev`, but I'm almsot sure you ain't gonna need the modificators

Reply:
okay) done